src/modules/code-block/code-block.component.ts

```diff
@@ -53,10 +53,8 @@ export class StacheCodeBlockComponent implements AfterViewInit {
     this.cdRef.detectChanges();
   }
 
-  public getClassNames(): string {
-    if (this.languageType) {
-      return `language-${this.languageType}`;
-    }
+  public getClassName(): string {
+    return `language-${this.languageType}`;
```

Comment:
so if no languageType gets passed in,  it will return `language-undefined` ?   or is there a default?